docs/content/drafts/PageLayout.mdx

```diff
@@ -0,0 +1,181 @@
+---
+title: PageLayout
+status: Draft
+description: TODO
+---
+
+<Note variant="warning">Not implemented yet</Note>
+
+## Examples
+
+### Default
+
+```jsx
+<PageLayout>
+  <PageLayout.Header>Header</PageLayout.Header>
+  <PageLayout.Content>Content</PageLayout.Content>
+  <PageLayout.Pane>Pane</PageLayout.Pane>
+  <PageLayout.Footer>Footer</PageLayout.Footer>
+</PageLayout>
+```
+
+### With dividers
+
+```jsx
+<PageLayout>
+  <PageLayout.Header divider="line">Header</PageLayout.Header>
+  <PageLayout.Content>Content</PageLayout.Content>
+  <PageLayout.Pane divider="line">Pane</PageLayout.Pane>
+  <PageLayout.Footer divider="line">Footer</PageLayout.Footer>
+</PageLayout>
+```
+
+### Condensed
+
+```jsx
+<PageLayout columnGap="condensed" rowGap="condensed">
+  <PageLayout.Header>Header</PageLayout.Header>
+  <PageLayout.Content>Content</PageLayout.Content>
+  <PageLayout.Pane>Pane</PageLayout.Pane>
+  <PageLayout.Footer>Footer</PageLayout.Footer>
+</PageLayout>
+```
+
+## Props
+
+### PageLayout
+
+<!-- TODO: Resoponsive variants -->
+
+<PropsTable>
+  <PropsTableRow
+    name="containerWidth"
+    type={`| 'full'
+| 'medium'
+| 'large'
+| 'xlarge'`}
+    defaultValue="'full'"
+    description="Define the maximum width of the page container."
+  />
+  <PropsTableRow
+    name="outerSpacing"
+    type={`| 'normal'
```

Comment:
I'm fine with t-shirt sizing names for these cc @vdepizzol

Reply:
@colebemis I think `condensed` and `normal` do a better job at describing their semantic usage for this variable. In the case of `normal`, it provides a better hint that it's the default one, whether `condensed` is specific to hm, _condensed_ UI experiences.

I think the t-shirt sizing names make sense in a lot of scenarios, but are too generic for this situation since all their values come with the same weight.